frontend/src/components/pages/admin/UsersPage.tsx

```diff
@@ -131,19 +123,12 @@ const UsersPage = (): React.ReactElement => {
               <TabPanel>
                 <VStack pt={4} spacing={6}>
                   <HStack width="100%">
-                    <InputGroup width="95%">
-                      <Input
-                        borderRadius="6px"
-                        borderColor="grey.100"
-                        backgroundColor="grey.100"
-                        onChange={(e) => setSearch(e.target.value)}
-                        placeholder="Search bar"
-                      />
-                      <InputRightElement pointerEvents="none" h="full">
-                        <SearchOutlineIcon />
-                      </InputRightElement>
-                    </InputGroup>
-                    <SortTablePopover OrderingSets={OrderingSets} />
+                    <SearchBar onSearch={setSearch} />
+                    <SortMenu
+                      tableProperties={["firstName", "email"]}
```

Comment:
In the next PR I think tableProperties will have the same value as headers in the AdminUserTable (and AdminUserTable can be removed as a separate file.